### PR TITLE
[docs] getting started

### PR DESCRIPTION
**Pull request by jamiltz**

### CODE REVIEW

File: Swift/Tests/SampleCodeTest.swift [hunk 1]
```diff
@@ -536,5 +536,80 @@ class SampleCodeTest: CBLTestCase {
...
+        }
+        
+        // Create a new document (i.e. a record) in the database.
+        let mutableDoc = MutableDocument()
+            .setFloat(2.0, forKey: "version")
+            .setString("SDK", forKey: "type")
+        
+        // Save it to the database.
+        do {
+            try database.saveDocument(mutableDoc, concurrencyControl: .lastWriteWins)
```
Comment (pasin) [inline Swift/Tests/SampleCodeTest.swift]: For simplicity, should we call the default database.save(mutableDoc) instead? Do you intentionally want to call with concurrencyControl parameter?

File: Swift/Tests/SampleCodeTest.swift [hunk 2]
```diff
@@ -536,5 +536,80 @@ class SampleCodeTest: CBLTestCase {
...
+        } catch {
+            fatalError("Error saving document")
+        }
+        
+        // Update a document.
+        if let mutableDoc = database.document(withID: mutableDoc.id)?.toMutable() {
+            mutableDoc.setString("Swift", forKey: "language")
+            do {
+                try database.saveDocument(mutableDoc, concurrencyControl: .lastWriteWins)
```
Comment (pasin) [inline Swift/Tests/SampleCodeTest.swift]: For simplicity, should we call the default database.save(mutableDoc) instead?

File: Swift/Tests/SampleCodeTest.swift [hunk 3]
```diff
@@ -536,5 +536,80 @@ class SampleCodeTest: CBLTestCase {
...
+        let replConfig = ReplicatorConfiguration(database: database, target: targetEndpoint)
+        replConfig.replicatorType = .pushAndPull
+        
+        // Add authentication.
+        replConfig.authenticator = BasicAuthenticator(username: "john", password: "pass")
+        
+        // Create replication.
+        let replication = Replicator(config: replConfig)
+        
+        // Listen to database change events.
```
Comment (pasin) [inline Swift/Tests/SampleCodeTest.swift]: Should be Listen to replicator change event?

File: Swift/Tests/SampleCodeTest.swift [hunk 4]
```diff
@@ -536,5 +536,80 @@ class SampleCodeTest: CBLTestCase {
...
+        // Create replicators to push and pull changes to and from the cloud.
+        let targetEndpoint = URLEndpoint(url: URL(string: "ws://localhost:4984/example_sg_db")!)
+        let replConfig = ReplicatorConfiguration(database: database, target: targetEndpoint)
+        replConfig.replicatorType = .pushAndPull
+        
+        // Add authentication.
+        replConfig.authenticator = BasicAuthenticator(username: "john", password: "pass")
+        
+        // Create replication.
+        let replication = Replicator(config: replConfig)
```
Comment (pasin) [inline Swift/Tests/SampleCodeTest.swift]: I think the variable name should be replicator to match the class name?

File: Swift/Tests/SampleCodeTest.swift [hunk 5]
```diff
@@ -536,5 +536,80 @@ class SampleCodeTest: CBLTestCase {
...
+        
+        // Create replicators to push and pull changes to and from the cloud.
+        let targetEndpoint = URLEndpoint(url: URL(string: "ws://localhost:4984/example_sg_db")!)
+        let replConfig = ReplicatorConfiguration(database: database, target: targetEndpoint)
+        replConfig.replicatorType = .pushAndPull
+        
+        // Add authentication.
+        replConfig.authenticator = BasicAuthenticator(username: "john", password: "pass")
+        
+        // Create replication.
```
Comment (pasin) [inline Swift/Tests/SampleCodeTest.swift]: Create Replicator?

File: docs/modules/ROOT/pages/objc.adoc [hunk 6]
```diff
@@ -62,6 +62,39 @@ Cocoapods.
...
+include::{examplesdir}/SampleCodeTest.m[tags=getting-started]
+----
+
+Build and run. You should see the document ID and property printed to
+the console. The document was successfully persisted to the database.
+
+image:getting-started-ios.png[image]
+
+Before synchronizing documents to Sync Gateway you will need to disable
+App Transport Security. In the Xcode navigator, right-click on
```
Comment (pasin) [inline docs/modules/ROOT/pages/objc.adoc]: Is it more clear to also describe why the App Transport Security need to disable?

**Comment by jamiltz** [comment]
@pasin thanks for the review. I updated the Swift getting started and mention of App Transport Security. The ObjC getting started code will need to be added before this PR can be merged.

**Comment by jamiltz** [comment]
@pasin I've added the ObjC version. If that code looks fine I think this PR can be merged.

**Comment by jamiltz** [comment]
The commits on this PR were moved to https://github.com/couchbase/couchbase-lite-ios/pull/2125. Closing.

**Pull request closed without merging**